### PR TITLE
Mongo reactive spring

### LINKED ISSUE #3: Netty classloading issue
Solve Netty classloading issue caused by fabric intermediate remapped vanilla which includes netty
conflicting with spring reactor netty (which has a different version). Knot class loader
is picking de intermediate vanilla classes causing a NoSuchMethodError

### PR DESCRIPTION
**Pull request by AdanJoGoHe**
Resolves #3, #5

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,104 +1,129 @@
-plugins {
-	id 'fabric-loom' version '0.9.+'
-	id 'maven-publish'
-	id 'org.springframework.boot' version '2.5.3'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
+import com.github.jengelman.gradle.plugins.shadow.transformers.PropertiesFileTransformer
 
+plugins {
+    id 'fabric-loom' version '0.9.+'
```
Comment (adelfuns) [inline build.gradle:4]: This plugins versions should be in a properties file too

File: build.gradle [hunk 2]
```diff
@@ -1,104 +1,129 @@
...
-		credentials(HttpHeaderCredentials) {
-			name = 'Private-Token'
-			value = project.git_lab_private_token
-		}
-		authentication {
-			header(HttpHeaderAuthentication)
-		}
-	}
+    mavenCentral()
+    maven {
```
Comment (adelfuns) [inline build.gradle:32]: This now needs to be deleted, no longer working

File: build.gradle [hunk 3]
```diff
@@ -1,104 +1,129 @@
...
-	compileOnly 'org.projectlombok:lombok:1.18.20'
-	annotationProcessor 'org.projectlombok:lombok:1.18.20'
-	implementation 'dev.miku:r2dbc-mysql:0.8.2.RELEASE'
-	implementation 'org.springframework.boot:spring-boot-starter-data-r2dbc:2.5.3'
-	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    minecraft "com.mojang:minecraft:${project.minecraft_version}"
+    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+    modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
+    modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
```
Comment (adelfuns) [inline build.gradle:71]: Jackson version

File: build.gradle [hunk 4]
```diff
@@ -1,104 +1,129 @@
...
+                builtBy remapJar
+            }
+            artifact(sourcesJar) {
+                builtBy remapSourcesJar
+            }
+        }
+    }
+
+    repositories {
+        maven {
```
Comment (adelfuns) [inline build.gradle:117]: This now needs to be deleted, no longer working

File: src/main/java/net/eriagacha/controller/GachaTelemetryController.java [hunk 5]
```diff
@@ -12,17 +12,22 @@
...
-  public GachaTelemetryController() {
-  }
+  private GachaTelemetryController() {}
 
 
   public static void InsertTelemetry(String playerName, String rewardObtained) {
     GachaTelemetryRepository gr =
         EriaGachaMain.springContext.getBean(GachaTelemetryRepository.class);
 
+    /*
```
Comment (adelfuns) [inline src/main/java/net/eriagacha/controller/GachaTelemetryController.java:26]: Delete this comment

File: src/main/java/net/eriagacha/controller/GachaTelemetryController.java [hunk 6]
```diff
@@ -40,7 +48,12 @@ public static void InsertTelemetry(String playerName, String rewardObtained) {
                 name.getDate()))
         .flatMap(gr::save);
 
-    saved.subscribe(log::info);
+    try {
+      saved.subscribe();
+    }catch (Exception e){
+      log.error("Erorr de GTC" + e.getMessage());
```
Comment (adelfuns) [inline src/main/java/net/eriagacha/controller/GachaTelemetryController.java:54]: Message in spanish and what does GTC mean?

File: src/main/resources/application.properties [hunk 7]
```diff
@@ -1,3 +1 @@
-spring.r2dbc.url=r2dbc:mysql://virtual-eria.com:3306/eriacraft
-spring.r2dbc.username=eriacraft
-spring.r2dbc.password=RCc<Vy1pc?M=&<&y^
+spring.data.mongodb.uri=mongodb://eriacraft:RCc<Vy1pc?M=&<&y^@virtual-eria.com:3306/eriacraft
```
Comment (adelfuns) [inline src/main/resources/application.properties:1]: We need to check how to make secret of this passwords

**Review by adelfuns — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (adelfuns) [reply build.gradle:32]: See #15

File: build.gradle (hunk 4 above)
Comment (adelfuns) [reply build.gradle:117]: See #15

File: src/main/resources/application.properties (hunk 7 above)
Comment (adelfuns) [reply src/main/resources/application.properties:1]: see #16

File: build.gradle (hunk 1 above)
Comment (AdanJoGoHe) [reply build.gradle]: Done

File: build.gradle (hunk 3 above)
Comment (AdanJoGoHe) [reply build.gradle]: Done

File: src/main/java/net/eriagacha/controller/GachaTelemetryController.java (hunk 5 above)
Comment (AdanJoGoHe) [reply src/main/java/net/eriagacha/controller/GachaTelemetryController.java:26]: Done

File: src/main/java/net/eriagacha/controller/GachaTelemetryController.java (hunk 6 above)
Comment (AdanJoGoHe) [reply src/main/java/net/eriagacha/controller/GachaTelemetryController.java:54]: Done

**Review by adelfuns — APPROVED** [review]

**Pull request merged**